doc/README.adoc

```diff
@@ -14,7 +14,10 @@ Consider an example of troubleshooting an Alert on the OCP console.
 == Opening the panel
 
 First open the alert of interest in the console.
-Now open the troubleshooting panel [FIXME screenshot of global button ]
+Now open the troubleshooting panel, using the "Signal Correlation" entry in the troubleshooting menu, at top right of the screen:
```

Comment:
This sounds like it is required to go to the Alerting page to use the global opener. Could we adjust to wording to show that the alerting page has a dedicated button for it and that there is also a global button to open the panel?

Reply:
Good feedback, check out the update. I've made it clearer that Alerts is just an example, and also included instructions so the user can re-produce the example for themselves.

Reply:
I have deliberately not mentioned the old alert-specifc or log-specific links as I think we will want to get rid of them now we have a uniform way of using the panel across resource types.

Reply:
We already have a couple of stories ([OU-478](https://issues.redhat.com/browse/OU-478) [OU-479](https://issues.redhat.com/browse/OU-479)) to get rid of them.